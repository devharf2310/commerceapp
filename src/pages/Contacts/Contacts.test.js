import React from 'react'
import renderer from 'react-test-renderer'
import Contacts from './Contacts';

describe('contacts snapshot tests', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Contacts/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
