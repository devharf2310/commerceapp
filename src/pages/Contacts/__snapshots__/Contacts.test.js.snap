// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`contacts snapshot tests renders correctly 1`] = `
<h1>
  Contacts
</h1>
`;
